importers/trading212/dividends_importer.py

```diff
@@ -0,0 +1,54 @@
+from dividend_payment import DividendTransaction
+import csv
+import decimal
+
+FUND_DOMICILE_MAPPING = {
+    "IWQU": "Ireland",
+    "IUSN": "Ireland",
+    "IQQW": "Ireland",
+    "VAGF": "Ireland",
+    "IS3S": "Ireland",
+    "IQQH": "Ireland",
+}
+
+FUND_NAME_MAPPING = {
```

Comment:
This `FUND_NAME_MAPPING` is not used. I think it is probably copied from another file but was forgotten.